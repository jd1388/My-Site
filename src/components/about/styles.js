import styled from 'styled-components';

import Colors from '../../styles/Colors';

export const Hero = styled.h1`
    font-size: 90pt;
    color: ${Colors.blue};
    font-weight: 400;
    padding-top: 15vh;
    margin-bottom: 0;
    align-self: center;

    @media (min-width: 800px) {
        font-size: 160pt;
        padding-top: 20vh;
    }
`;

export const AboutContainer = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 0 10%;
    height: 95vh;
    min-height: 800px;
    font-family: Roboto, sans-serif;

    @media (min-width: 800px) {
        padding: 0 25%;
        min-height: 1000px;
    }
`;

export const SectionContent = styled.div`
    text-align: justify;
    font-weight: 300;
    color: ${Colors.black};
    flex-grow: 1;
    display: flex;
    align-items: center;
`;
