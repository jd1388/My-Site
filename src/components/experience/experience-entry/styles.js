import styled from 'styled-components';

import Colors from '../../../styles/Colors';

export const ExperienceEntryContainer = styled.div`
    
    margin-bottom: 32px;
    display: grid;
    grid-template-columns: 1fr;


    &:last-child {
        margin-bottom: 0;
    }

    @media (min-width: 800px) {
        grid-template-columns: 30% 5% 65%;
    }
`;

export const ExperienceEntryHeader = styled.h4`
    font-weight: normal;
    color: ${Colors.white};
    text-align: left;
    font-size: 12pt;
    margin: 0;
    grid-column: 1;
    margin-bottom: 8px;

    @media (min-width: 800px) {
        font-size: 16pt;
    }
`;

export const ExperienceEntryContent = styled.p`
    font-weight: 300;
    text-align: justify;
    margin: 0;
    grid-column: 1;
    font-size: 12pt;

    @media (min-width: 800px) {
        grid-column: 3;
        font-size: 16pt;
    }
`;

export const ExperienceEntryDates = styled(ExperienceEntryContent)`
    grid-column: 1;
    text-align: left;
    margin-bottom: 8px;

    @media (min-width: 800px) {
        grid-row: 1;
        margin-bottom: 0;
    }
`;

export const ExperienceEntryPosition = styled(ExperienceEntryHeader)`
    grid-row: 2;

    @media (min-width: 800px) {
        grid-column: 1;
    }
`;
