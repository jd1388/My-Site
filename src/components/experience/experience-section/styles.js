import styled from 'styled-components';
import Colors from '../../../styles/Colors';

export const ExperienceSectionHeader = styled.h3`
    font-weight: 300;
    color: ${Colors.white};
    font-size: 18pt;

    @media (min-width: 800px) {
        font-size: 22pt;
    }
`;
