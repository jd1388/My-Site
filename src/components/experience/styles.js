import styled from 'styled-components';

import Colors from '../../styles/Colors';

export const ExperienceContainer = styled.div`
    background-color: ${Colors.blue};
    width: 100%;
    min-height: 95vh;
    font-family: Roboto, sans-serif;
    color: ${Colors.white};
    display: flex;
    justify-content: center;
    align-items: center;
    box-shadow: 0px 4px 20px 0px rgba(0,0,0,0.75);
    transform: skewY(2deg);
    padding: 5vh 0;
    filter: url('data:image/svg+xml;utf8,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><filter id="gaussian_blur"><feGaussianBlur in="SourceGraphic" stdDeviation="0" /></filter></defs></svg>#gaussian_blur');
`;

export const ExperienceContent = styled.div`
    padding: 50px 10%;
    transform: skewY(-2deg);

    @media (min-width: 800px) {
        padding: 50px 25%;
    }
`;

export const ExperienceHeader = styled.h2`
    font-weight: normal;
    color: ${Colors.white};
    text-align: center;
    margin: 0;
    font-size: 36pt;

    @media (min-width: 800px) {
        font-size: 48pt;
    }
`;
