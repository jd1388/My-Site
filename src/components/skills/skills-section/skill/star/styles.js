import styled from 'styled-components';
import StarIcon from 'mdi-react/StarIcon';
import StarOutlineIcon from 'mdi-react/StarOutlineIcon';

import Colors from '../../../../../styles/Colors';

export const StyledStarIcon = styled(StarIcon)`
    fill: ${Colors.blue};
`;

export const StyledStarOutlineIcon = styled(StarOutlineIcon)`
    fill: ${Colors.blue};
`;
