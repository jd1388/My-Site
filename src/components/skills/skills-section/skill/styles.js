import styled from 'styled-components';

export const SkillEntryContainer = styled.div`
    display: flex;
    justify-content: space-between;
    padding-bottom: 5px;
    width: 100%;

    &:nth-child(3n - 1) {
        grid-column-start: 3;
    }

    &:nth-child(3n) {
        grid-column-start: 5;
    }
`;

export const SkillEntryHeader = styled.span`
    font-size: 15pt;
    font-weight: 300;
`;
