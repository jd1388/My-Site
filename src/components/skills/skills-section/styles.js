import styled from 'styled-components';

import Colors from '../../../styles/Colors';

export const SkillSectionHeader = styled.h3`
    font-weight: 300;
    font-size: 18pt;
    color: ${Colors.blue};

    @media (min-width: 800px) {
        font-size: 22pt;
    }
`;

export const SkillSectionEntries = styled.div`
    display: flex;
    flex-wrap: wrap;

    @media (min-width: 800px) {
        display: grid;
        grid-template-columns: 30% 1fr 30% 1fr 30%;
    }
`;
