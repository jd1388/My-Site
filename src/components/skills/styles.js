import styled from 'styled-components';

import Colors from '../../styles/Colors';

export const SkillsContainer = styled.div`
    align-items: center;
    background-color: ${Colors.white};
    color: ${Colors.black};
    display: flex;
    font-family: Roboto, san-serif;
    justify-content: center;
    width: 100%;
`;

export const SkillsContent = styled.div`
    padding: 50px 10%;
    min-width: 80%;

    @media (min-width: 800px) {
        padding: 50px 25%;
        min-width: 970px;
    }
`;

export const SkillsHeader = styled.h2`
    color: ${Colors.blue};
    font-size: 36pt;
    font-weight: normal;
    margin: 0;
    text-align: center;

    @media (min-width: 800px) {
        font-size: 48pt;
    }
`;
