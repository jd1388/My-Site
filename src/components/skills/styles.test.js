import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { SkillsContainer, SkillsContent, SkillsHeader } from './styles';

describe('Skills Styles', () => {
    test('renders the SkillsContainer correctly', () => {
        const wrapper = shallow(<SkillsContainer />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('renders the SkillsContent correctly', () => {
        const wrapper = shallow(<SkillsContent />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('renders the SkillsHeader correctly', () => {
        const wrapper = shallow(<SkillsHeader />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
