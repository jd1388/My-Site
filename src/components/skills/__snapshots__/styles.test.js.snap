// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills Styles renders the SkillsContainer correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    align-items: center;
    background-color: ",
          "#FFFFFF",
          ";
    color: ",
          "#232323",
          ";
    display: flex;
    font-family: Roboto, san-serif;
    justify-content: center;
    width: 100%;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Skills Styles renders the SkillsContent correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": false,
        "rules": Array [
          "
    padding: 50px 10%;
    min-width: 80%;

    @media (min-width: 800px) {
        padding: 50px 25%;
        min-width: 970px;
    }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Skills Styles renders the SkillsHeader correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-htpNat",
        "isStatic": false,
        "rules": Array [
          "
    color: ",
          "#0984E3",
          ";
    font-size: 36pt;
    font-weight: normal;
    margin: 0;
    text-align: center;

    @media (min-width: 800px) {
        font-size: 48pt;
    }
",
        ],
      },
      "displayName": "styled.h2",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-htpNat",
      "target": "h2",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
